include/picongpu/plugins/common/openPMDWriteMeta.hpp

```diff
@@ -105,6 +105,10 @@ namespace picongpu
                  * optional metadata:
                  */
 
+                //  PIConGPU is writing particles based on the openPMD standard 'ED-PIC' extension.
```

Comment:
Also the field data is in line with the ED-PIC extension of openPMD. I guess just the comment is wrong.